src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -325,6 +319,31 @@ export class StudyViewPageStore {
 
     @observable showComparisonGroupUI = false;
 
+    @observable private _NAValueSet = observable.map<boolean>();
```

Comment:
Should we reuse existing `_clinicalDataBinFilterSet` and extend `ClinicalDataBinFilter` with `showNA` property instead of creating a new observable?

Reply:
I think that is a great idea, since _clinicalDataBinFilterSet also uses a uniqueKey. I missed that!